### ISSUE TITLE
Does netopeer2 server support listening in a VRF?

### ISSUE THREAD

**Issue by nickbroon** [CESNET/netopeer2#1099]
Can netopeer2 be natively configured to listen in a VRF? (That is to bind it's listening socket to a VRF device)

https://stackoverflow.com/questions/30207581/is-there-a-way-to-set-a-socket-option-to-indicate-which-vrf-to-use
https://manpages.debian.org/testing/iproute2/ip-vrf.8.en.html

**Comment by michalvasko** [comment]
No, there is no VRF-specific code.

**Comment by nickbroon** [comment]
Thanks for confirming. My quick look over the code and yang model had me suspecting as much.
Consider this an enhancement request :-)

**Comment by michalvasko** [comment]
Fair enough but it does not exactly seem like a simple feature to support and since we have no use-case for it, it will not be implemented anytime soon.
